src/editor/CSSInlineEditor.js

```diff
@@ -151,20 +151,12 @@ define(function (require, exports, module) {
         return html;
     }
 
-    /**
-     * This function is registered with EditManager as an inline editor provider. It creates a CSSInlineEditor
-     * when cursor is on an HTML tag name, class attribute, or id attribute, find associated
-     * CSS rules and show (one/all of them) in an inline editor.
-     *
-     * @param {!Editor} editor
-     * @param {!{line:Number, ch:Number}} pos
-     * @return {?$.Promise} synchronously resolved with an InlineWidget; or error
-     *         {string} if pos is in tag but not in tag name, class attr, or id attr; or null if the
-     *         selection isn't even close to a context where we could provide anything.
-     */
-    function htmlToCSSProvider(hostEditor, pos) {
+    function checkProvider(hostEditor){
+        var selectorResult = {result : undefined};
+        return (validateProvider(hostEditor,selectorResult) === true) ? true : false;
```

Comment:
In JS you don't need to make this a ternary operation, just cast the return value of `validateProvider` to a boolean:

```js
return !!validateProvider(hostEditor,selectorResult)
```